src/game/shell.c

```diff
@@ -194,18 +271,40 @@ bool Shell_MakeScreenshot()
         break;
     }
 
-    char path[20];
-    for (int i = 0; i < 10000; i++) {
-        sprintf(path, "screenshot%04d.%s", i, ext);
+    // Screenshot folder
+    const char ss_folder[SS_FOLDER_NAME_SIZE] = "screenshots";
+    char *full_ss_path = NULL;
+    File_GetFullPath(ss_folder, &full_ss_path);
+    File_CreateDirectory(full_ss_path);
+    Memory_FreePointer(&full_ss_path);
+
+    // Screenshot name
+    char ss_name[TIMESTAMP_SIZE + LEVEL_TITLE_SIZE];
+    Shell_GetScreenshotName(ss_name);
+
+    // Screenshot folder/name path
+    char path
+        [SS_FOLDER_NAME_SIZE + TIMESTAMP_SIZE + LEVEL_TITLE_SIZE
+         + EXTENSION_SIZE];
+    sprintf(path, "%s/%s.%s", ss_folder, ss_name, ext);
+    char *full_path = NULL;
+    File_GetFullPath(path, &full_path);
```

Comment:
Now it's not needed to `File_GetFullPath` for the second time, we can just `char path[strlen(full_ss_path) + ...]; sprintf(...);` and not have to `Memory_FreePointer` again.